packages/babel-plugin-proposal-decorators/src/transformer-2021-12.ts

```diff
@@ -0,0 +1,946 @@
+import type { NodePath } from "@babel/traverse";
+import { types as t } from "@babel/core";
+import syntaxDecorators from "@babel/plugin-syntax-decorators";
+import ReplaceSupers from "@babel/helper-replace-supers";
+import * as charCodes from "charcodes";
```

Comment:
Can you edit https://github.com/babel/babel/blob/ddd93b9c61a197711c2e3eae8a8b175783ea6976/babel.config.js#L197-L199 to also inline `charCodes` in this file?